src/Features/Customer/Dashboard/index.js

```diff
@@ -1,42 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import { Grid, Typography, Box } from '@material-ui/core';
-import { makeStyles } from '@material-ui/core/styles';
 import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+import { useDispatch } from 'react-redux';
 
 import MainTab from '../../../components/CardMenus/Tabs';
+import TemporaryDrawer from '../../../components/Drawer';
 import SideMenu from '../../../components/sideMenu';
+import { Header, OrderHeading, OrderRef, OrderDetails, UserInfo, FirstTab } from './style';
 
 function Dashboard() {
-  const useStyles = makeStyles(() => ({
-    header: {
-      display: 'flex',
-      justifyContent: 'space-between',
-      marginTop: '30px',
-    },
-    orderHeading: {
-      fontWeight: '700',
-      marginLeft: '40px',
-    },
-    orderRef: {
-      color: 'lightgrey',
-      paddingLeft: '10px',
-      paddingTop: '8px',
-      fontSize: '18px',
-    },
-    orderDetails: {
-      display: 'flex',
-    },
-    userInfo: {
-      paddingRight: '40px',
-    },
-    tabs: {
-      paddingRight: '30px',
-    },
-  }));
-  const classes = useStyles();
+  const [isDrawerOpen, setOpenDrawer] = useState(false);
 
-  const { header, orderDetails, orderHeading, orderRef, userInfo, tabs } = classes;
+  const dispatch = useDispatch();
+  const [value, setValue] = useState(0);
+
+  function toggleDrawer() {
+    setOpenDrawer(!isDrawerOpen);
+  }
+
+  const openDrawer = () => {
+    setOpenDrawer(true);
+  };
+
+  const closeDrawer = () => {
```

Comment:
why do we need to write explicit function for open and close drawer, while we have toggleDrawer???